test/e2e/testbackend/main.go

```diff
@@ -0,0 +1,184 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Why not use https://github.com/api7/grpc_server_example?

Reply:
I think we can keep it like this for now